mentions/src/test/kotlin/io/spine/examples/pingh/mentions/MentionSpec.kt

```diff
@@ -88,8 +94,8 @@ public class MentionSpec : ContextAwareTest() {
         }
 
         @Test
-        public fun `set 'SNOOZED' status in 'Mention' entity`() {
-            val expected = Mention::class.buildBy(id, MentionStatus.SNOOZED)
+        public fun `set 'SNOOZED' status and set 'snooze_until_when' field in 'Mention' entity`() {
```

Comment:
Can we describe scenarios is a domain-specific way, rather than stating the changes in field values?

It won't be possible to do everywhere, but here it surely is:

```
snooze the target `Mention`, remembering the time until which it is snoozed
```

The reason is that the fields may get renamed. Even the messages may get renamed (but not as likely, as fields). But the scenario remains the same.

Please review the display names of other tests as well.